uber/site_sections/registration.py

```diff
@@ -502,6 +502,19 @@ def lost_badge(self, session, id):
         raise HTTPRedirect('index?message={}', 'Badge has been recorded as lost.')
 
     @ajax
+    def check_out_attendee_in_error(self, session, **params):
+        message = "That ID was not found."
+        if params.get('id'):
+            a = session.query(Attendee).filter(Attendee.id == params['id']).first()
+            if a:
```

Comment:
I'm personally okay raising an exception if we somehow get an invalid or missing id, since that means something has gone horribly wrong.  The client can always pass an error handler to their ``$.ajax`` call if they want to display a nice ``toastr`` message in that case.

If we go that route, and implement my suggested approach below, then this entire method becomes

```python
@ajax
def check_out_attendee_in_error(self, session, id):
    attendee = session.attendee(id)
    attendee.checked_in = None
    if attendee.badge_type not in c.PREASSIGNED_BADGE_TYPES:
        attendee.badge_num = None
    session.commit()
    return {
        'new_badge': attendee.badge,
        'message': 'Attendee un-checked-in.'
    }
```